cmd/minikube/cmd/start.go

```diff
@@ -194,10 +193,9 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Bool(skipRegistryCheck, false, "Skip Docker daemon registry check")
 	clusterUpFlagSet.String(publicHostname, "", "Public hostname for OpenShift cluster")
 	clusterUpFlagSet.String(routingSuffix, "", "Default suffix for server routes")
-	clusterUpFlagSet.Bool(useExistingConfig, false, "Use existing configuration if present")
-	clusterUpFlagSet.String(hostConfigDir, dockerhost.DefaultConfigDir, "Directory on Docker host for OpenShift configuration")
+	clusterUpFlagSet.String(hostConfigDir, "/var/lib/minishift/openshift.local.config", "Directory on Docker host for OpenShift configuration")
 	clusterUpFlagSet.String(hostVolumesDir, dockerhost.DefaultVolumesDir, "Directory on Docker host for OpenShift volumes")
-	clusterUpFlagSet.String(hostDataDir, "", "Directory on Docker host for OpenShift data. If not specified, etcd data will not be persisted on the host.")
+	clusterUpFlagSet.String(hostDataDir, "/var/lib/minishift/hostdata", "Directory on Docker host for OpenShift data. If not specified, etcd data will not be persisted on the host.")
```

Comment:
This one (`/var/lib/minishift/hostdata`) too need to be added in const or variable.